packages/components/src/components/drawer/drawer.stories.ts

```diff
@@ -0,0 +1,276 @@
+          }
+        ]
+      })}
+    </div>`;
+  }
+};
+
+export const Mouseless = {
+  parameters: { controls: { exclude: ['default', 'header', 'footer', 'contained', 'no-header'] } },
+  render: (args: any) => {
+    return html`<div class="mouseless" id="header-with-btn" style="width: auto; height: 95vh; position: relative;">
+      <style>
+        #header-with-btn sd-drawer::part(header) {
+          padding-left: 0;
+        }
+      </style>
+      ${generateTemplate({
+        args,
+        constants: [
+          {
+            type: 'slot',
+            name: 'default',
+            value: `<input placeholder="Default slot"></input>`
+          },
+          {
+            type: 'slot',
+            name: 'header',
+            value: `<sd-button slot='header' variant='tertiary'>Header</sd-button>`
+          },
+          {
+            type: 'slot',
+            name: 'footer',
+            value: `<sd-button slot='footer' variant='secondary'>Footer</sd-button>`
+          },
+          {
+            type: 'attribute',
+            name: 'no-header',
+            value: false
+          }
+        ]
+      })}
+    </div>`;
+  },
+  play: async ({ canvasElement }: { canvasElement?: HTMLUnknownElement }) => {
+    const el = canvasElement?.querySelector('.mouseless sd-drawer');
+    await waitUntil(() => el?.shadowRoot?.querySelector('header'));
+
+    if (el?.shadowRoot) {
+      const header = el.shadowRoot.querySelector('header');
+      if (header) {
+        await userEvent.type(header, '{space}', { pointerEventsCheck: 0 });
+      }
+    }
+  }
+};
```

Comment:
The mouseless story should include a button which opens the drawer and then tabs until it focuses the close button.

This makes it sure that people can open and close it.